src/ansys/fluent/visualization/graphics/graphics_windows.py

```diff
@@ -40,17 +40,14 @@ class _GraphicsWindow:
     save, animate, etc. on graphics objects.
     """
 
-    def __init__(self, grid: tuple = (1, 1)):
+    def __init__(
+        self, wind_id, grid: tuple = (1, 1), renderer=None, graphics_objs=None
```

Comment:
@prmukherj 
I would not recommend trying to abbreviate words like window this way. This is less readable than using the entire word (and not only because wind is another word with two unrelated meanings).

Reply:
Thank you @seanpearsonuk, it was a typo, for both plotter and graphics I had kept it "win_id", however I have updated to "window_id" in both places now.
Thank you.